custom/icds_reports/utils/aggregation.py

```diff
@@ -1828,7 +1828,8 @@ def aggregation_query(self):
             ('num_anc_complete', 'case_list.num_anc_complete'),
             ('valid_visits', 'agg_cf.valid_visits + agg_bp.valid_visits + agg_pnc.valid_visits'),
             ('opened_on', 'case_list.opened_on'),
-            ('dob', 'case_list.dob')
+            ('dob', 'case_list.dob'),
+            ('closed', 'case_list.closed')
```

Comment:
This should be using `closed_on` instead of `closed`. As this is written, it is getting the current value of closed instead of the historical value. This means that if we were trying to aggregate September data, but the case was closed in October, then we would save closed = true to September as well. To fix this, save `closed_on` and change the filters to be `closed_on < start_of_month` or `closed_on < end_of_month` depending on the intended outcome

Also there is already a column `open_in_month` in this table. Is there a reason that cannot be used?

Reply:
Right, using open_in_month is better. I have changed that.